src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/CodeStyle/VisualBasicCodeStyleOptions.vb

```diff
@@ -53,10 +53,22 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeStyle
                 editorConfigName:="visual_basic_style_unused_value_assignment_preference",
                 defaultValue:=New CodeStyleOption2(Of UnusedValuePreference)(UnusedValuePreference.UnusedLocalVariable, NotificationOption2.Suggestion),
                 optionsBuilder:=s_allOptionsBuilder)
+
+        Public Shared ReadOnly ParenthesesForInvocations As [Option2](Of CodeStyleOption2(Of Boolean)) = CreateOption(
+            VisualBasicCodeStyleOptionGroups.Parentheses, NameOf(ParenthesesForInvocations),
+            defaultValue:=CodeStyleOptions2.FalseWithSuggestionEnforcement, 'Suggestion or silent??
+            EditorConfigStorageLocation.ForBoolCodeStyleOption("visual_basic_style_add_parentheses_for_invocations")) ' RoamingProfileStorageLocation ??
```

Comment:
> visual_basic_style_parentheses_for_empty_method_invocations
> visual_basic_style_parentheses_for_empty_property_invocations

Reply:
> 
> 
> > visual_basic_style_parentheses_for_empty_method_invocations
> > visual_basic_style_parentheses_for_empty_property_invocations

I think these names are a bit unclear. The option for empty method adds parentheses but for properties it removes parentheses. But the names doesn't show that. What do you think?

Reply:
> The option for empty method adds parentheses but for properties it removes parentheses. 

i don't think it shoudl work that way.  i think the options should just be true/false around if parens are preferred or eschewed.  So we just have:

`visual_basic_style_parentheses_for_empty_method_invocations=true:silent` and
`visual_basic_style_parentheses_for_empty_property_invocations=false:silent`

Reply:
Ah got it :)